dledger/pom.xml

```diff
@@ -24,7 +24,7 @@
     </parent>
     <modelVersion>4.0.0</modelVersion>
 
-    <artifactId>dledger</artifactId>
+    <artifactId>dledger-core</artifactId>
```

Comment:
It would be better not to change the pom coordinates to ensure compatibility for upgrades.